### DISCUSSION TITLE
Feature Request: 拖拽功能和联合 node

### DISCUSSION THREAD

**Discussion by ajaxpost** [ueberdosis/tiptap#5819] [Feature Requests]
### Description

1. 在 ul 或者 ol 列表中，拖拽功能希望可以指定某个 li 元素进行拖拽，现在只能对整体进行一个拖拽。
2. 希望可以新增联合 node模式，比如 在 list 列表中可以插入标题元素 =》 - # 112

### Use Case

拖拽功能拓展

### Type

New feature

**Comment by ajaxpost** [comment]
@nperez0111 希望可以接受我的建议

**Reply by nperez0111** [reply]
I appreciate the feedback here, we hope to improve the drag-handle extension in the future. At this time though, we do not have plans to add this right now.
